code/src/main/java/com/onlinecontacttracing/servlets/authentication/CheckForApiAuthorizationServlet.java

```diff
@@ -0,0 +1,125 @@
+package com.onlinecontacttracing.authentication;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Collections;
+import java.util.List;
+
+import com.google.api.client.auth.oauth2.Credential;
+import com.google.api.client.googleapis.auth.oauth2.GoogleAuthorizationCodeFlow;
+import com.google.api.client.googleapis.auth.oauth2.GoogleClientSecrets;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdTokenVerifier;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken.Payload;
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.auth.oauth2.TokenResponse;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.client.auth.oauth2.AuthorizationRequestUrl;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.FileInputStream;
+import java.io.File;
+import java.security.GeneralSecurityException;
+import java.io.FileNotFoundException;
+import java.util.logging.Logger;
+
+/**
+*  This class directs the creation of user Credentials for accessing APIs.
+*/
+
+@WebServlet("/check-for-api-authorization")
+public abstract class CheckForApiAuthorizationServlet extends HttpServlet {
+
+  abstract void useCredential(Credential credential);
+  // should return "check-for-api-authorization" for now
```

Comment:
Say what the function should be doing conceptually, the fact that it hard-codes a value currently is irrelevant.